test

```diff
@@ -50,8 +50,8 @@ USERPKG=${PKG:-}
 COVER=${COVER:-"-cover"}
 
 # Hack: gofmt ./ will recursively check the .git directory. So use *.go for gofmt.
-IGNORE_PKGS="(vendor/|etcdserverpb|rafttest|gopath.proto|v3lockpb|v3electionpb)"
-INTEGRATION_PKGS="(integration|tests/e2e|contrib|functional)"
+IGNORE_PKGS="(^|/)(vendor|etcdserverpb|rafttest|gopath.proto|v3lockpb|v3electionpb|c)(/|$)"
+INTEGRATION_PKGS="(^|/)(integration|tests/e2e|contrib|functional|[.])(/|$)"
```

Comment:
Thanks for fixing the regex matching to test full path parts only.

Reply:
Your welcome